cachengo-sso/templates/deployment.yaml.bak

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1beta2
```

Comment:
Did you intend to check-in this file?

Reply:
deployment.yaml.bak is not needed. Removed that file.